src/index.ejs

```diff
@@ -66,6 +66,7 @@
   data-cozy-icon-path="{{.IconPath}}"
   data-cozy-tracking="{{.Tracking}}"
   data-cozy-app-slug="{{.AppSlug}}"
+  data-cozy-subdomain-type="{{.SubDomain}}"
```

Comment:
At some point I wonder if we should not only have the key `data-cozy` where all the data from the stack would go.

This would help us not having to add a key everytime we need a new piece of data from the stack. Additionally, the keys must be conventional and coherent between apps to be correctly picked up by cozy-client or other libs relying on those values, so having the app developer write those keys (and potentially mispelling them) is a risk that could be reduced if there was only one key. The interface would only be between cozy-client and the stack.